src/librustc_lint/builtin.rs

```diff
@@ -1333,6 +1347,12 @@ impl EarlyLintPass for EllipsisInclusiveRangePatterns {
             };
         }
     }
+
+    fn check_pat_post(&mut self, _cx: &EarlyContext<'_>, pat: &ast::Pat) {
+        if let Some(node_id) = self.node_id {
+            if pat.id == node_id { self.node_id = None }
```

Comment:
```suggestion
            if pat.id == node_id {
                self.node_id = None;
            }
```
(Blocks should be on new lines.)